### PR TITLE
Add basic augmentation

### PR DESCRIPTION
**Pull request by sarahmakki12**
Added basic augmentation class.

### CODE REVIEW

File: models/UNetModule.py [hunk 1]
```diff
@@ -2,6 +2,24 @@
 import tensorflow as tf
 from tensorflow import keras
 
+class Augment(tf.keras.layers.Layer):
+  def __init__(self, seed=42):
+    super().__init__()
+    # both use the same seed, so they'll make the same random changes.
+    self.augment_inputs = keras.Sequential([
+      tf.keras.layers.RandomFlip(mode="horizontal", seed=seed),
```
Comment (pijner) [inline models/UNetModule.py]: Is there a benefit to doing horizontal and vertical sequentially vs using `mode="horizontal_and_vertical"`?

File: models/UNetModule.py
Comment (pijner) [inline models/UNetModule.py:1]: Should the output of the sementation model be 1 channel since we're predicting a bitmap? We could do 4 channel if we decide to do rgba.

File: models/UNetModule.py (hunk 1 above)
Comment (sarahmakki12) [reply models/UNetModule.py]: No there isn't, I changed it to `mode="horizontal_and_vertical"`.

File: models/UNetModule.py
Comment (sarahmakki12) [reply models/UNetModule.py:1]: Yeah I mixed up the output channels vs the number of classes, it's fixed now. For the current data it's set to 2 classes.

**Review by pijner — APPROVED** [review]
Looks good. I'll be adding a losses module containing some weighted losses. I'll move the dice loss function there
